ui/component/viewers/videoViewer/internal/chapters.jsx

```diff
@@ -184,6 +184,8 @@ export function parseAndLoad(player: any, claim: StreamClaim) {
     overrideHoverTooltip(player, tsData, duration);
   } else {
     deleteHoverInformation(player);
+    // $FlowIssue
+    player?.controlBar?.getChild('ChaptersButton')?.hide();
```

Comment:
:question: Now, when player loads, it will always flash the chapters button before the AutoPlay-Next or Rate button appears.  Not ideal, but better than current production behavior of retaining the chapters button.
- [ ] Next, when I cast, the chapters button comes back, displaying chapters from a prior video. We can probably ship this incremental update first and come back to Chromecast, though (up to you)
  - <img width="450" alt="image" src="https://user-images.githubusercontent.com/64950861/176158430-32bcdb18-af69-438e-aa27-3409f5ae3e52.png">